runtests.py

```diff
@@ -445,7 +445,7 @@ def build_project(args):
         os.makedirs(site_dir)
     if not os.path.exists(site_dir_noarch):
         os.makedirs(site_dir_noarch)
-    env['PYTHONPATH'] = site_dir + ':' + site_dir_noarch
+    env['PYTHONPATH'] = site_dir + os.pathsep + site_dir_noarch
```

Comment:
This mistake makes me suspicious - do we really need to set `PYTHONPATH` here? It is done correctly before running tests.